cookbook/core/control_flow/subworkflows.py

```diff
@@ -59,20 +49,17 @@ def my_subwf(a: int = 42) -> Tuple[str, str]:
     u, v = t1(a=x)
     return y, v
 
-
 # %%
-# Example 1:
-# ^^^^^^^^^^^
-# This is the parent workflow. In it, we call the workflow declared above.
-# This also showcases how to override the node name of a task (or subworkflow). Typically, nodes are just named
+# We call the workflow declared above in the parent workflow below. 
+# It showcases how to override the node name of a task (or subworkflow). Typically, nodes are just named
 # sequentially, ``n0``, ``n1``, and so on. Because the inner ``my_subwf`` also has a ``n0`` you may
-# wish to change the name of the first one. Not doing so is also fine - Flyte will automatically prepend something
+# wish to change the name of the first one. Not doing so is also fine - Flyte automatically prepends an attribute
 # to the inner ``n0``, since node IDs need to be distinct within a workflow graph. This issue does not exist
-# when calling something by launch plan since those launch a separate execution entirely.
+# when calling entities by launch plan since they create a separate execution entirely.
```

Comment:
Its already there in external flow- "When launch plans are used within a workflow to launch an execution of a previously defined workflow, a new external execution is launched, with a separate execution ID and can be observed as a distinct entity in FlyteConsole/FlyteCTL etc. ". I think it can be removed from here, right? @samhita-alla

Reply:
We can remove it.